internal/keyrotation/rotate_test.go

```diff
@@ -46,7 +46,9 @@ func TestRotateKeys(t *testing.T) {
 	key, aad, wrapped := testMakeKey(ctx, t, kms, keyID)
 
 	neg20Days, _ := time.ParseDuration("-480h")
+	neg40Days, _ := time.ParseDuration("-480h")
```

Comment:
neg20 and neg40 have the same value

Reply:
interestingly it still passes (and should) even if the gap is 0. The previous key was still effective for more than two weeks.